docs/modules/clusters/pages/ucn-member-side.adoc

```diff
@@ -105,10 +108,11 @@ user-code-namespaces:
     - id: MyJARsInZIPFileID
       resource-type: JARs-in-ZIP
```

Comment:
I'm not sure this is correct - I think for the yaml it should be JARS_IN_ZIP  (see https://github.com/hazelcast/hazelcast-mono/blob/master/hazelcast/hazelcast/src/main/java/com/hazelcast/internal/config/YamlMemberDomConfigProcessor.java#L344C3-L353C22) 

@JamesHazelcast second opinion?

Reply:
Yes, good catch @gareth-johnston